starknet_devnet/blueprints/base.py

```diff
@@ -221,3 +221,29 @@ async def fork_status():
     if config.fork_network:
         return jsonify({"url": config.fork_network.url, "block": config.fork_block})
     return jsonify({})
+
+
+@base.route("/abort_blocks_after", methods=["POST"])
+async def abort_blocks_after():
+    """Abort blocks and transactions from given block hash to last block."""
+    request_json = request.json or {}
+    current_block = await state.starknet_wrapper.blocks.get_by_hash(
```

Comment:
Shouldn't it be called `starting_block`?

Reply:
What happens if it's an invalid hash? What if it's a hash of an already aborted block?

Reply:
Fixed in 48eeb54bb5b0f00028e79eb19013862f83a070a2

With invalid hash StarknetErrorCode.BLOCK_NOT_FOUND exception will be raised. 

Wit calling the endpoint twice, the operation will be repeated, and there is no additional check if this is what you asking for.

Reply:
I think that this can be resolved